forge/files/src/index.js

```diff
@@ -42,6 +44,12 @@ app.on('window-all-closed', () => {
 });
 
 app.on('ready', () => {
+  if (isDev) {
+    installExtension(EMBER_INSPECTOR)
+      .then((name) => console.log(`Added Extension:  ${name}`))
```

Comment:
I don't think we really need to be logging this on every startup...just catch/console.error should be sufficient for logging, right?

Reply:
I had just copied the logs from https://github.com/MarshallOfSound/electron-devtools-installer#usage but I'm fine with removing them if you think we should.

Reply:
Yeah, it looks like it's just put there to make the usage more clear in the sample code -- I don't think we provide any value by telling the user each time they launch that we successfully installed the extension, and it will certainly annoy some users (at least one so far 😉)